obfx_modules/policy-notice/init.php

```diff
@@ -0,0 +1,265 @@
+<?php
+/**
+ * A module to check changes before theme updates.
+ *
+ * @link       https://themeisle.com
+ * @since      1.0.0
+ *
+ * @package    Theme_Update_Check_OBFX_Module
+ */
+
+/**
+ * The class defines a new module to be used by Orbit Fox plugin.
+ *
+ * @package    Theme_Update_Check_OBFX_Module
+ * @author     Themeisle <friends@themeisle.com>
+ */
+class Policy_Notice_OBFX_Module extends Orbit_Fox_Module_Abstract {
+
+	/**
+	 * Test_OBFX_Module constructor.
+	 *
+	 * @since   1.0.0
+	 * @access  public
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->name        = __( 'Policy Notice', 'themeisle-companion' );
+		$this->description = __( 'A simple notice bar which will help you inform users about your website policy.', 'themeisle-companion' );
+	}
+
+	/**
+	 * Method to determine if the module is enabled or not.
+	 *
+	 * @since   1.0.0
+	 * @access  public
+	 * @return bool
+	 */
+	public function enable_module() {
+		return true;
+	}
+
+	/**
+	 * The method for the module load logic.
+	 *
+	 * @since   1.0.0
+	 * @access  public
+	 * @return mixed
+	 */
+	public function load() {
+		return;
+	}
+
+	/**
+	 * Method that returns an array of scripts and styles to be loaded
+	 * for the front end part.
+	 *
+	 * @since   1.0.0
+	 * @access  public
+	 * @return array
+	 */
+	public function public_enqueue() {
+		return array();
+	}
+
+	/**
+	 * Method that returns an array of scripts and styles to be loaded
+	 * for the admin part.
+	 *
+	 * @since   1.0.0
+	 * @access  public
+	 * @return array
+	 */
+	public function admin_enqueue() {
+
+		return array();
+	}
+
+	/**
+	 * Method to define the options fields for the module
+	 *
+	 * @since   1.0.0
+	 * @access  public
+	 * @return array
+	 */
+	public function options() {
+
+		return array(
+			array(
+				'id'      => 'enable_policy_notice',
+				'name'    => 'enable_policy_notice',
+				'title'   => '',
+				'type'    => 'toggle',
+				'label'   => sprintf(
+					'%s <a href="%s">%s</a>.',
+					esc_html__( 'Allow OrbitFox to display a bottom bar with info about the website Private Policy. You can set the Private Policy page ', 'textdomain' ),
+					admin_url( 'privacy.php' ),
+					esc_html__( 'here', 'textdomain' )
+				),
+				'default' => '1',
```

Comment:
please set the default value as disabled in order to avoid activating this feature by mistake on module activation.